### PR TITLE
Add alert before creating a new problem

### PR DESCRIPTION
**Pull request by Aritra8438**
# Description

This PR adds an alert saying that `All the stored data will be lost` when they create a new problem.

https://github.com/user-attachments/assets/86bffacc-3ff3-41db-829c-a75649cd300b

# Checklist:

- [x] The code follows the [coding guidelines](https://github.com/omegaup/omegaup/wiki/Coding-guidelines) of omegaUp.
- [x] The tests were executed and all of them passed.
- [x] If you are creating a feature, the new tests were added.
- [x] If the change is large (> 200 lines), this PR was split into various Pull Requests. It's preferred to create one PR for changes in controllers + unit tests in PHPUnit,  and then another Pull Request for UI + tests in Jest, Cypress or both.
Signed-off-by: Aritra Majumder <aritramajumder8438@gmail.com>

### CODE REVIEW

**Comment by Aritra8438** [comment]
@carlosabcs, PTAL!

**Review by carlosabcs — APPROVED** [review]

**Comment by Aritra8438** [comment]
@heduenas, PTAL!

File: frontend/templates/en.lang [hunk 1]
```diff
@@ -1067,6 +1067,10 @@ problemCreatorCodeSave = "Save"
 problemCreatorCodeUpload = "Or upload the file"
 problemCreatorCodeUploadFilesize = "The file size limit for uploaded files is %(limit)"
 problemCreatorContentPlaceHolder = "Content"
+problemCreatorCreateNewProblem = "Create new problem"
+problemCreatorCreateNewProblemBack = "Back"
+problemCreatorCreateNewProblemContinue = "Continue"
+problemCreatorCreateNewProblemWarning = "Do you want to start again? All the previous data will be lost and the creator will be reset. Save the current progress before creating a new one. To do so, click on the button 'Generate problem'."
```
Comment (carlosabcs) [inline frontend/templates/en.lang:1073]: Does this work? If so, make the change for all the languages.
```suggestion
problemCreatorCreateNewProblemWarning = "Do you want to start again? All the previous data will be lost and the creator will be reset. Save the current progress before creating a new one. To do so, click on the button \"Generate problem\"."
```

**Review by carlosabcs — COMMENTED** [review]
Just a minor comment that I forgot to mention.

**Comment by Aritra8438** [comment]
Done! @carlosabcs

**Review by heduenas — APPROVED** [review]

**Pull request merged**